### PR TITLE
Feature/sc-25305/implement-clark-reports-route-in-clark-gateway

### PR DESCRIPTION
**Pull request by bereketsega**
This PR adds `clark-reports` route to `clark-Gateway`.

### CODE REVIEW

**Review by mfranz1 — APPROVED** [review]

**Review by shanioffe — APPROVED** [review]

File: src/modules/clark-reports/ClarkReportsController.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { Controller } from "../../interfaces/Controller";
+import { REPORTS_ROUTES } from "../../routes";
+
+const CLARK_REPORTS_API = process.env.CLARK_REPORTS_LAMBDA_URI;
+
+export class ClarkReportsController implements Controller {
+  buildRouter(): Router {
+    const router = Router();
+
+    router
```
Comment (Cwagne17) [inline src/modules/clark-reports/ClarkReportsController.ts:12]: We should add Swagger docs comment block on top of this route.

**Review by Cwagne17 — COMMENTED** [review]
Just a reminder to add block comment for swagger docs. Check out the other routes in the Gateway for an example.

**Review by JosueMolinaMorales — APPROVED** [review]
Before we deploy this, we need to add the `CLARK_REPORTS_LAMBDA_URI` env to the infra

File: src/modules/clark-reports/ClarkReportsController.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+     * @swagger
+     * /reports:
+     * patch:
+     *    description: Get all reports
+     *    tags:
+     *      - Clark Reports
+     *    responses:
+     *        200:
+     *            description: OK
+     *        400:
```
Comment (Cwagne17) [inline src/modules/clark-reports/ClarkReportsController.ts:22]: What would a 400 be a result of?

Comment (bereketsega) [reply src/modules/clark-reports/ClarkReportsController.ts:22]: It would be for Missing email or name in body

**Pull request merged**